src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -135,8 +138,7 @@ public class TokenAcquisition : ITokenAcquisition
             }
             catch (MsalException ex)
             {
-                // brentsch - todo, write to a log
-                Debug.WriteLine(ex.Message);
+                _logger.LogInformation(ex.Message);
```

Comment:
LogInformation? or LogDebug?

Reply:
@jmprieur i'm not sure. See [#8 here](https://nblumhardt.com/2017/07/library-logging/) this is why i put the others in Debug, but this is an exception.